src/LM/langmodel.jl

```diff
@@ -0,0 +1,246 @@
+abstract type Langmodel end
+abstract type gammamodel <: Langmodel end #BaseNgram with Add-one smoothing algo
+abstract type InterpolatedLanguageModel <: Langmodel end #Interpolated language model with smoothing
+
+#DataType MLE
+#Type for providing MLE ngram model scores.
+#Implementation of Base Ngram Model.
+
+struct MLE <: Langmodel
+    vocab::Vocabulary
+end
+
+"""
+    MLE(word::Vector{T}, unk_cutoff=1, unk_label="<unk>") where { T <: AbstractString}
+
+Initiate Type for providing MLE ngram model scores.
+
+Implementation of Base Ngram Model.
+   
+"""
+function MLE(word, unk_cutoff=1, unk_label="<unk>")
```

Comment:
`word` needs a type tag here. Since other parameters of this function have default value, it can revert to a one argument form, that is then ambigous with the struct definiton. Is `word` meant to be a vector? The docstring suggests so, and the function definition should be amended

Reply:
yes word is Vector here
I will make a small PR with all word tags